TinyMCE.Blazor/wwwroot/TinyMce.Blazor.lib.module.js

```diff
@@ -1,5 +1,24 @@
 console.log('loading js tinymce-blazor');
 
+const isTinyMceVersionLessThan = (major, minor) =>
+  (typeof tinymce !== 'undefined' && (Number(tinymce.majorVersion) < major || Number(tinymce.minorVersion.split('.')[0]) < minor));
+
+const hasDisabledSupport = () => !isTinyMceVersionLessThan(7, 6);
```

Comment:
It is better to check whether `disabled` option is registered like [this approach in vue integration](https://github.com/tinymce/tinymce-vue/blob/main/src/main/ts/Utils.ts#L158)